ShowPot/ShowPot/Common/UI/SPDefaultBottomSheetViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  SPDefaultBottomSheetViewController.swift
+//  ShowPot
+//
+//  Created by Daegeon Choi on 8/5/24.
+//
+
+import UIKit
+import SnapKit
+import Then
+import RxSwift
+import RxRelay
+
+class SPDefaultBottomSheetViewController: BottomSheetViewController {
+    
+    // MARK: Property
+    let message: String
+    let buttonTitle: String
+    
+    let didTapBottomButton = PublishRelay<Void>()
+    private let disposeBag = DisposeBag()
+    
+    // MARK: UI Component
+    
+    lazy var messageLabel = SPLabel(KRFont.H1).then { label in
+        label.textColor = .gray000
+        label.numberOfLines = 0
+        label.setText(message)
+    }
+    
+    lazy var bottomButton = SPButton(.accentBottomEnabled).then { button in
+        button.setText(buttonTitle)
+    }
```

Comment:
```suggestion
    lazy var bottomButton = SPButton(.accentBottomEnabled).then { button in
        button.setText(buttonTitle)
        button.configuration?.background.cornerRadius = 2
    }
```
- bottomButton에 cornerRadius값을 2로 수정해야할꺼같아요

Reply:
이부분은 `SPButton.accentBottomEnabled`에 지정된 스타일을 수정해야 하는 부분입니다.
다른 지정 스타일에도 cornerRadius 빠져있어서 추가했습니다